webaudio/the-audio-api/the-convolvernode-interface/convolver-response-4-chan.html

```diff
@@ -14,6 +14,8 @@
       // Test various convolver configurations when the convolver response has
       // a four channels.
 
+      const absoluteThreshold = 3 * Math.pow(2, -22);
```

Comment:
Is this empirical? Can we say so, or provide a rationale for picking this value and multiplier?